src/main/java/world/bentobox/chat/commands/island/IslandChatCommand.java

```diff
@@ -30,11 +30,9 @@ public void setup() {
 
     @Override
     public boolean canExecute(User user, String label, List<String> args) {
-        boolean is = false;
 
-        if(is = this.getIslands().getIslandAt(user.getLocation()).isPresent())
-            island = this.getIslands().getIslandAt(user.getLocation()).get();
-        return is;
+        island = this.getIslands().getIslandAt(user.getLocation()).orElse(null);
+        return island != null;
```

Comment:
return this.getIslands().getIslandAt(user.getLocation()).isPresent()

Reply:
Hmm, but I use the `private @Nullable Island island;` in other places below canExecute() method, shall I #get it later in execute(), or it's fine to #get it from here too?

Reply:
@wellnesscookie You are doing the right thing. If the result is null, then you return false. If the result is and island, then you get the island and return true.

Reply:
Oh, you are assigning value to class variable... yea.., that is the reason why I always use this. before class variables... so it could be viewed from git change log.